### PR TITLE
Add types (issue 189)

### PR DESCRIPTION
**Pull request by allieychen**
Add types annotation for the project. This pull request completes [this project](https://github.com/googlegenomics/gcp-variant-transforms/projects/5).

Issues: [189](https://github.com/googlegenomics/gcp-variant-transforms/issues/189)

### CODE REVIEW

File: gcp_variant_transforms/options/variant_transform_options.py [hunk 1]
```diff
@@ -126,6 +128,7 @@ def add_arguments(self, parser):
               'optimize_for_large_inputs are enabled.'))
 
   def validate(self, parsed_args, client=None):
+    # type: (argparse.Namespace) -> None
```
Comment (allieychen) [inline gcp_variant_transforms/options/variant_transform_options.py]: I didn't find any function calls pass `client` except the unit tests. I wonder whether we can remove it.

File: gcp_variant_transforms/libs/annotation/vep/vep_runner.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def _get_api_request_fixed_parts(self):
     }
 
   def _make_action(self, *args, **kwargs):
-    # type: (str, str) -> Dict
+    # type: (*str, **str) -> Dict
```
Comment (allieychen) [inline gcp_variant_transforms/libs/annotation/vep/vep_runner.py]: @bashir2 Can you help check this method? There is a warning here. It seems that it is passing a List instead a dictionary.

Comment (bashir2) [reply gcp_variant_transforms/libs/annotation/vep/vep_runner.py]: So like we discussed, the correct type is `**List[str]`, thanks for fixing this.

Comment (allieychen) [reply gcp_variant_transforms/libs/annotation/vep/vep_runner.py]: Updated. Thanks!

File: gcp_variant_transforms/options/variant_transform_options.py (hunk 1 above)
Comment (arostamianfar) [reply gcp_variant_transforms/options/variant_transform_options.py]: I think unit tests is the main use case for it actually. We should keep it. Could you add the type for it?

**Pull request merged**